.github/workflows/main.yml

```diff
@@ -182,7 +182,7 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - name: Check Single Commit
-        uses: actions/github-script@v5.1.0
+        uses: actions/github-script@v6
```

Comment:
We must tag the most specific version number in order to review dependabot bumps from a minor or patch releases to the next release.  `v6.0.0` is the most specific version, but `v6` is too general.  Our builds would silently bump from the first v6 version to the next v6 minor or patch version without any dependabot PR review until v7 came out.  Arguably this is a dependabot bug for choosing `v6` when it had a choice of two tags, but please change the tag to `v6.0.0` before you merge.

Reply:
Actually, I'm making an assumption that dependabot wouldn't bump v6 again until v7 came out.  I don't know that for a fact, but is it worth the bother to find out :-)?

Reply:
Ah yeah, good catch. Updated.